pkg/kubelet/certificate/transport.go

```diff
@@ -105,18 +106,20 @@ func addCertRotation(stopCh <-chan struct{}, period time.Duration, clientConfig
 				// the certificate has been deleted from disk or is otherwise corrupt
 				if now.After(lastCertAvailable.Add(exitAfter)) {
 					if clientCertificateManager.ServerHealthy() {
-						klog.Fatalf("It has been %s since a valid client cert was found and the server is responsive, exiting.", exitAfter)
+						klog.ErrorS(nil, "It has been sometime since a valid client cert was found and the server is responsive, exiting.", "exitAfter", exitAfter)
```

Comment:
We could maybe improve this message a little, maybe with something like:

```
klog.ErrorS(nil, "Shutting down as no valid client cert was found and the server was responsive", "lastCertificateAvailabilityTime", lastCertAvailable, "shutdownThreshold", exitAfter)
```

Reply:
Thanks, the code has been updated @endocrimes